recipes/pymc3/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pymc3" %}
+{% set version = "3.0rc2" %}
+{% set md5 = "80105941758b9eba47c8a2774d5ee905" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    # - numpy >=1.11.0
+    # - scipy >=0.12.0
+    # - matplotlib >=1.5.0
+    # - theano >=0.8.2
+    # - pandas >=0.18.0
+    # - patsy >=0.4.0
+    # - joblib >=0.9
+    # - tqdm >=4.8.4
+    # - enum43 # [py2k]
```

Comment:
Do not do it in this PR. Wait for the creation of the feedstock.

I merged this to avoid clogging the CIs here. Also, you will have merge rights in the feedstock, so you won't need to wait for me to merge it :smile: